internal/serve/httperror/errors_test.go

```diff
@@ -8,18 +8,22 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/stellar/wallet-backend/internal/apptracker"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 )
 
 func TestErrorResponseRender(t *testing.T) {
+	appTrackerMock := apptracker.MockAppTracker{}
+	appTrackerMock.On("CaptureException", mock.Anything).Return(nil)
```

Comment:
You are not testing the code here. It would be best if you did something like:

```go
appTrackerMock.On("CaptureException", errors.New("Some error"))
defer appTrackerMock.AssertExpectations(t)
```

I'm pretty sure that we can omit the `Return` since the capture exception doesn't return errors. Additionally, we should call the `AssertExpectations` method.

Then we should change the test a little:

```go
in: *InternalServerError(context.Background(), "", errors.New("Some error"), nil, &appTrackerMock),
```

Reply:
Done